src/pl/michaeli/service/ConfirmationResource.java

```diff
@@ -0,0 +1,220 @@
+package pl.michaeli.service;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.core.GenericEntity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+import org.springframework.context.annotation.AnnotationConfigApplicationContext;
+
+import pl.michaeli.model.BookedSeat;
+import pl.michaeli.model.ErrorMessage;
+import pl.michaeli.model.Reservation;
+import pl.michaeli.model.Seat;
+import pl.michaeli.spring.config.AppConfig;
+import pl.michaeli.spring.dao.BookedSeatsDAO;
+import pl.michaeli.spring.dao.ReservationsDAO;
+import pl.michaeli.spring.dao.ScreeningsDAO;
+import pl.michaeli.spring.dao.SeatsDAO;
+import pl.michaeli.spring.dao.TicketTypesDAO;
+
+
+@Path("confirm")
+public class ConfirmationResource {
+
+	@POST
+	@Consumes({MediaType.APPLICATION_XML,MediaType.APPLICATION_JSON})
+    public Response confirmReservation(Reservation reservation) {
+
+		String name = reservation.getName();
+		String surname = reservation.getSurname();
+		int screeningId = reservation.getScreeningId();
+		List<BookedSeat> seatsToBook = reservation.getBookedSeats();
+
+		if(name.length()<3 || surname.length()<3) {
+			ErrorMessage errorMessage = new ErrorMessage("Za kr�tkie imi� lub nazwisko!");
+            return Response.serverError().type(MediaType.APPLICATION_JSON).entity(errorMessage).build();
+		}
+		if(checkName(name)==false) {
+			ErrorMessage errorMessage = new ErrorMessage("B��dne imi�!");
+            return Response.serverError().type(MediaType.APPLICATION_JSON).entity(errorMessage).build();
+		}
+		if(checkSurname(surname)==false) {
+			ErrorMessage errorMessage = new ErrorMessage("B��dne nazwisko!");
```

Comment:
Dużo tego kodu się powtarza, może dałoby się to łatwiej napisać?

Reply:
Lekko poprawiłam. Jest lepiej?

Reply:
Ciupinkę, chociaż nad tym można by jeszcze mocno popracować :)